tools/snapshot/CMakeLists.txt

```diff
@@ -0,0 +1,66 @@
+# Build snapshot tool
+# Need to be installed before building: go protoc-gen-go protoc-gen-go-grpc
+
+set(GO_EXECUTABLE "go")
```

Comment:
Maybe instead of doing this we should create a go package exporting libxrpl's proto file and use the package here. I think it would be more user friendly to be able to build this tool with simply `go build`, not this conan-cmake-go C++ style way.

Reply:
I think it is more friendly to use conan-cmake-go way, since it is inside Clio repo. It has the same way of building clio_server.

Reply:
While it's a good idea to have this separated out - let's not spend time doing this now and instead keep it in the repo until we feel we really need to separate this tool. I agree with Cindy that it's more convenient and maintainable to do it this way now.